prow/gerrit/adapter/adapter.go

```diff
@@ -325,3 +341,21 @@ func (c *Controller) ProcessChange(instance string, change client.ChangeInfo) er
 
 	return nil
 }
+
+func shouldTriggerOnMessage(lastUpdate time.Time, change client.ChangeInfo, presubmit config.Presubmit) bool {
```

Comment:
We just spent a lot of time making this simpler and I think this PR just makes it more complicated. I think we should start with the "why"